src/set_theory/ordinal/basic.lean

```diff
@@ -922,6 +922,8 @@ def omega : ordinal.{u} := lift $ @type ℕ (<) _
 
 localized "notation `ω` := ordinal.omega" in ordinal
 
+@[simp] theorem type_nat_lt : @type ℕ (<) _ = ω := (lift_id _).symm
```

Comment:
Just to check I understand this correct, it actually states
```suggestion
@[simp] theorem type_nat_lt : @type ℕ (<) _ = ω.{0} := (lift_id _).symm
```
right?

Reply:
Yes.

Reply:
I think it might be worth writing that explicitly for clarity